new-packaging/bin/build-rpm-package

```diff
@@ -82,6 +84,7 @@ then
 fi
 
 # Copy manpages
+rm -rf ${sources_directory}/manpages
```

Comment:
Just a nice-to-have. If you already have built something then the files' presence means `gzip` will stop on an interactive prompt asking if you want to override. Couldn't see a force/yes argument so I added this to avoid the prompt.